### ISSUE TITLE
New Logo for Shoes

### ISSUE THREAD

**Issue by JesseHerrick** [shoes/shoesrb.com#53]
This goes along with issue #52. How does everyone feel about a new logo for Shoes?

**Comment by PragTob** [comment]
I'd so love it. E.g. a logo for shoes4, we were searching for one on the mailing list a year or more back or so... entries weren't overwhelming and in the end the vote ended in sticking with the old logo.

Totally up to a new logo - maybe featuring 4 shoes as I figured the reason why shoes3 had 3 is that the version number was 4 :)

**Comment by JesseHerrick** [comment]
@PragTob I was thinking something more minimalist. Modern logos seem to be going that way.

**Comment by PragTob** [comment]
@JesseHerrick was just an idea, open to about anything - you are right 4 is probably a bit much and doesn't scale well. Would want it to be kind of playful though :)

**Comment by wpp** [comment]
@JesseHerrick Can't wait to see what you come up with. <3

**Comment by davorb** [comment]
@wpp @JesseHerrick You and me both.

**Comment by nathancarnes** [comment]
Is there still interest in this? I'd be happy to pull together a few concepts.

**Comment by JesseHerrick** [comment]
I would definitely say so. I'm not as good at graphic design as I am web
design.

**Comment by PragTob** [comment]
DEFINITELY! As we move forward to a release candidate a new logo sure would be nice :)

**Comment by jasonrclark** [comment]
FWIW, I commissioned a new logo for Shoes 4 from @traviseugeneclark, and the resulting assets for that are available in a [new repo on this org](https://github.com/shoes/shoes-assets).
